manifests/init.pp

```diff
@@ -67,6 +68,18 @@
 
   $powershell_filename = regsubst($url, '^(.*\/)(.+?)(?:\.[^\.]*$|$)$', '\2')
 
+
+  if $proxyAddress {
+    warning("${module_name}: The use of proxyAddress in Download_file[${title}] is deprecated. Use proxy_address instead.")
```

Comment:
notify{} will be detected as a change in state every time, which is probably not what we want. I'm ambivalent...